js/localgov-publications.js

```diff
@@ -15,3 +15,54 @@
     }
   };
 })(jQuery, Drupal);
+
+/**
+   * Collapse able menu for Publication page
+   *
+   * @param {object} context
+   */
+Drupal.behaviors.publicationMenuToggle = {
+  attach: function(context) {
+    const menuCollapseBreakpoint = 992;
```

Comment:
Is this value correct for every LGD site? If not, it'll need to be configurable somehow, or done another way.